wiki/en/Directories.md

```diff
@@ -1,22 +1,22 @@
 ---
 layout: wiki
-title: "Custom Directories"
+title: "Directories"
 lang: "en"
-permalink: "/wiki/Custom-Directories"
+permalink: "/wiki/Directories"
 ---
 
 {% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
 
 
-# Running a Custom Directory
+# Running a Directory
 
 This is a specialised Jamulus Server configuration, as described in [Server Types](Running-a-Server#server-types).
 
-To view Servers listed by a Custom Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Custom Directories field. Multiple addresses can be added in this way if needed. Custom Directories will then appear in the Directory drop-down list on their Connect window. Custom Directories otherwise work for Clients in the same way as Public Directories, displaying a list of Servers registered with them.
+To view Servers listed by a Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Directories field. Multiple addresses can be added in this way if needed. Directories will then appear in the Directory drop-down list on their Connect window. Directories otherwise work for Clients in the same way as the built-in Directories, displaying a list of Servers registered with them.
 
 To run a Server as a Directory, it should be started with the `--directoryserver` option to make itself (that is, `localhost` or `127.0.0.1`) the Directory to query for Servers.
 
-When running a Server with the GUI, set the Custom Directory server address in the Options tab to "localhost", then select "Custom" from the Directory drop-down list.  
+When running a Server with the GUI, set the Directory server address in the Options tab to "localhost", then select "Custom" from the Directory drop-down list. 
```

Comment:
Again, here, "Custom" is correct as it has to be entered by hand.  (Note that, even if the address is the same as one of the built in ones and it's entered by hand, it's "Custom".)